### PR TITLE
Various model print fixes

### LINKED ISSUE #5411: When a model is printed in Companion, the curve used in Limits page is not printed
When a model is printed in Companion, the curve used in Limits page is not printed.
This error occurs in OpenTx 2.1.9 and OpenTx 2.2.0 versions.

### PR DESCRIPTION
**Pull request by elecpower**
Fixes #5411 #3294
Rename Limits to Outputs for consistency with radio and Companion
Fix Subtrim, Min and Max gvars printed as values
Print PPM Center #1063 and Linear Subtrims

### CODE REVIEW

**Comment by elecpower** [comment]
Travis build environment crashed out

**Comment by 3djc** [comment]
I just relaunched it

**Comment by 3djc** [comment]
Sonar playing with us...

File: companion/src/modelprinter.cpp [hunk 1]
```diff
@@ -712,3 +712,52 @@ QString ModelPrinter::printGlobalVarPopup(int idx)
   return (model.gvarData[idx].popup ? "Y" : "N" );
 }
 
+QString ModelPrinter::printOutputValueGVar(int val)
+{
+  QString result = "";
+  if (abs(val) > 10000) {
+    if (val < 0) result = "-";
```
Comment (mpaperno) [inline companion/src/modelprinter.cpp]: Add line break here please (and another instance a few lines down).

**Comment by elecpower** [comment]
Travis build still failing...

**Pull request merged**